src/screens/Analytics/PaymentsAnalytics/PaymentAnalytics.res

```diff
@@ -101,34 +119,125 @@ let make = () => {
     actualdata->Array.map(Nullable.make)
   }
 
+  let title = "Payments Analytics"
+  let subTitle = "Gain Insights, monitor performance and make Informed Decisions with Payment Analytics."
+
+  let formaPayload = (singleStatBodyEntity: DynamicSingleStat.singleStatBodyEntity) => {
+    [
+      AnalyticsUtils.getFilterRequestBody(
+        ~filter=singleStatBodyEntity.filter,
+        ~metrics=singleStatBodyEntity.metrics,
+        ~delta=?singleStatBodyEntity.delta,
+        ~startDateTime=singleStatBodyEntity.startDateTime,
+        ~endDateTime=singleStatBodyEntity.endDateTime,
+        ~mode=singleStatBodyEntity.mode,
+        ~groupByNames=["currency"]->Some,
+        ~customFilter=?singleStatBodyEntity.customFilter,
+        ~source=?singleStatBodyEntity.source,
+        ~granularity=singleStatBodyEntity.granularity,
+        ~prefix=singleStatBodyEntity.prefix,
+        (),
+      )->JSON.Encode.object,
+    ]
+    ->JSON.Encode.array
+    ->JSON.stringify
```

Comment:
Please use LogicUtils.getJsonFromArrayOfJson

Reply:
`LogicUtils.getJsonFromArrayOfJson` takes the format of `[(string,JSON)]` but here its just `[(JSON)]`